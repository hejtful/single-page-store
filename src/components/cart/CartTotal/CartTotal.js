import React from 'react';

import styles from './CartTotal.module.css';

export const CartTotal = ({ amount }) => (
  <div className={styles.wrapper}>Total: {amount}$</div>
);
