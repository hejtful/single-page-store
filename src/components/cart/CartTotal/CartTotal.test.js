import React from 'react';
import { render } from '@testing-library/react';
import { CartTotal } from './CartTotal';

it('renders the total amount and currency sign', () => {
  const { container } = render(<CartTotal amount={1000} />);

  expect(container).toMatchSnapshot();
});
