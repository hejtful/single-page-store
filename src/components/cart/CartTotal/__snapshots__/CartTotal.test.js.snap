// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the total amount and currency sign 1`] = `
<div>
  <div
    class="wrapper"
  >
    Total: 
    1000
    $
  </div>
</div>
`;
